Astro Starter Kit: Toolbar App | Cannot find module 'node:url' or its corresponding type declarations.

### Astro Info

```block
Cannot find module 'node:url' or its corresponding type declarations.

1 import { fileURLToPath } from 'node:url';
                                ~~~~~~~~~~


Found 1 error in src/integration.ts:1

npm ERR! code 2
npm ERR! path /home/projects/gqairzpbmk.github
npm ERR! command failed
npm ERR! command sh -c npm run build
```


### If this issue only occurs in one browser, which browser is a problem?

_No response_

### Describe the Bug

When I open it via stackblitz it shows it

### What's the expected result?

Project preview

### Link to Minimal Reproducible Example

https://stackblitz.com/github/withastro/astro/tree/latest/examples/toolbar-app

### Participation

- [ ] I am willing to submit a pull request for this issue.

--- a/examples/toolbar-app/package.json
+++ b/examples/toolbar-app/package.json
@@ -15,6 +15,7 @@
     "./app": "./dist/app.js"
   },
   "devDependencies": {
+    "@types/node": "^18.17.8",
     "astro": "^5.1.2"
   }
 }

